wuzz.go

```diff
@@ -517,6 +561,47 @@ func (a *App) ToggleHistory(g *gocui.Gui, _ *gocui.View) error {
 	return nil
 }
 
+func (a *App) ToggleMethodlist(g *gocui.Gui, _ *gocui.View) error {
+	_, err := g.View("method-list")
+	if err == nil {
+		a.closeMethodlist(g)
+		return nil
+	}
+	g.Cursor = false
+	var method *gocui.View
+	maxX, maxY := g.Size()
+	height := len(METHODS)
+	if height > maxY-1 {
+		height = maxY - 1
+	}
+	width := 50
+	if width > maxX-4 {
+		width = maxX - 4
+	}
+	if method, err = g.SetView("method-list", maxX/2-width/2-1, maxY/2-height/2-1, maxX/2+width/2, maxY/2+height/2+1); err != nil {
+		if err != gocui.ErrUnknownView {
+			return nil
+		}
+		method.Wrap = false
+		method.Frame = true
+		method.Title = "Methods"
+		method.Highlight = true
+		method.SelFgColor = gocui.ColorYellow
```

Comment:
These view settings are the same as the history popup window's settings. Perhaps it could be extracted to a function (e.g. `setPopupViewDefaults()`) to avoid code duplication.